maquetado/scss/login.scss

```diff
@@ -0,0 +1,95 @@
+@import "colors";
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.login-container {
```

Comment:
Again, you can have just one class for this. Use a form tag here as wrapper and add this class.